libc/test/include/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_custom_target(libc_include_tests)
+
+add_libc_unittest(
+  sys_queue_test
+  SUITE
+    libc_include_tests
+  SRCS
+    sys/queue_test.cpp
+  DEPENDS
+    libc.include.llvm-libc-macros.sys_queue_macros
+    libc.src.__support.char_vector
+    libc.src.__support.CPP.string
+  COMPILE_OPTIONS
+    -Wno-gnu-statement-expression-from-macro-expansion
```

Comment:
Perhaps worth a comment //why// this is necessary. (I have no problem adding, and //I// know why, but future travelers might not know immediately, and I'm likely to forget).